assets/css/reponsive.css

```
@@ -0,0 +1,381 @@
+/* pc all */
+@media (min-width: 1024px) {
+  .toggle-menu {
+    display: none;
+  }
+  .navbar__icon-cart--count::before{
+    left: -8px;
+  }
+}
+/* mini PC */
+@media (min-width: 1024px) and (max-width: 1239px) {
+  .item-square {
+    padding: 34px;
+  }
+  .header__navbar-item {
+    padding: 5px;
+  }
+
+  .header__navbar__menu {
+    margin: 0 30px 0 5px;
+  }
+
+  .navbar__icon-cart--count::before{
+    left: -8px;
+  }
+}
+.toggle-menu {
+  display: none !important;
+}
+.navbar__icon-menu {
+  display: none;
+}
+
+/*tablet*/
+@media (min-width: 600px) and (max-width: 1024px) {
+  .footer__before {
+    padding: 60px 0;
+  }
+
+  .footer__list-item {
+    align-items: center;
+  }
+
+  .toggle-menu {
+    opacity: 0;
+  }
+  .navbar__icon-menu {
+    display: block;
+  }
+
+  .nav-icon {
+    display: inline-block;
+    margin-right: 20px;
+    gap: 10px;
+  }
+
+
+  .header__navbar {
+    display: flex;
+    align-items: center;
+  }
+
+  .header__navbar .nav-icon {
+    order: -1;
+  }
+
+  .navbar__icon-menu {
+    color: var(--white-color);
+    font-size: 24px;
+    cursor: pointer;
+    display: inline-block;
+    text-align: right;
+  }
+
+  .header__navbar__menu {
+    margin: 0;
+    padding: 0;
+    margin-bottom: 10px;
+  }
+
+  .header__navbar-item.active {
+    color: var(--white-color);
+    padding: 5px 15px;
+  }
+
+  .header__navbar-list {
+    display: none;
+    position: absolute;
+    right: 0px;
+    flex-direction: column;
+    background-color: #333;
+    width: 100%;
+    align-items: flex-end;
+    margin-top: 10px;
+  }
+
+  .header__navbar-item {
+    font-size: 16px;
+    padding: 10px 20px;
+    display: block;
+    color: #fff;
+    text-align: center;
+  }
+
+  #toggle-menu:checked ~ .header__navbar-list {
+    display: block;
+  }
+
+
+  .nav-icon-blue {
+    display: none;
+  }
+  .navbar___icon-language,
+  .navbar___icon-cart,
+  .nav-icon-blue {
+    margin: 0 5px;
+  }
+
+  .footer__after__copyright {
+    text-align: center;
+  }
+
+  .footer__after__icon{
+    justify-content: center;
+  }
+
+  .space{
+    margin: 60px 0 20px 0;
+  }
+
+  .slider__title-before {
+    font-size: 60px;
+  }
+
+  .slider__title-after {
+    font-size: 60px;
+  }
+
+  .content__paragraph__adblock {
+    font-size: 70px;
+  }
+
+  .img_word{
+    margin: 170px 0 0 0;
```

```suggestion
    margin-top: 170px;
```

Thay thế bằng class cha. xoá class này